src/.vuepress/integrations.json

```diff
@@ -268,130 +293,148 @@
+    "image": "amazon.svg",
+    "url": "https://www.myparcel.nl/en/integrations/amazon/"
   },
   {
     "name": "wix",
     "title": "Wix",
     "type": "api",
     "internal": true,
     "documentation": "/documentation/20.wix.html",
-    "image": "wix.svg"
+    "image": "wix.svg",
+    "url": "https://nl.wix.com/app-market/myparcel"
   },
   {
     "name": "ecwid",
     "title": "Ecwid",
     "type": "plugin",
     "internal": true,
     "documentation": "/documentation/17.ecwid.html",
-    "image": "ecwid.svg"
+    "image": "ecwid.svg",
+    "url": "https://www.ecwid.com/apps/shipping/myparcel"
   },
   {
     "name": "bol-com",
     "title": "bol.com",
     "type": "api",
     "internal": true,
     "documentation": "https://myparcelnl.github.io/bol.com",
-    "image": "bol-com.svg"
+    "image": "bol-com.svg",
+    "url": "https://partnerplatform.bol.com/nl/intermediair/myparcel/"
   },
   {
     "name": "exact-online",
     "title": "Exact Online",
     "type": "api",
     "internal": true,
     "documentation": "https://myparcelnl.github.io/exact-online",
-    "image": "exact-online.svg"
+    "image": "exact-online.svg",
+    "url": "https://www.myparcel.nl/koppelingen/exact-online-verzenden/"
   },
   {
     "name": "etsy",
     "title": "Etsy",
     "type": "api",
     "internal": true,
     "documentation": "/documentation/30.etsy.html",
-    "image": "etsy.svg"
+    "image": "etsy.svg",
+    "url": "https://www.myparcel.nl/koppelingen/etsy-verzenden/"
   },
   {
     "name": "picqer",
     "title": "Picqer",
     "type": "api",
     "documentation": "https://picqer.com/nl/help/artikelen/myparcel-koppelen",
-    "image": "picqer.svg"
+    "image": "picqer.svg",
+    "url": "https://picqer.com/nl/integraties/myparcel"
   },
   {
     "name": "channel-dock",
     "title": "ChannelDock",
     "type": "api",
     "documentation": "https://help.channeldock.com/books/nederlands-nl/page/myparcel-koppelen-aan-channeldock",
-    "image": "channel-dock.svg"
+    "image": "channel-dock.svg",
+    "url": "https://help.channeldock.com/books/nederlands-nl/page/myparcel-koppelen-aan-channeldock"
   },
   {
     "name": "goedgepickt",
     "title": "Goedgepickt",
     "type": "api",
     "documentation": "https://help.goedgepickt.nl/article/srujx2j0ln-my-parcel",
-    "image": "goedgepickt.svg"
+    "image": "goedgepickt.svg",
+    "url": "https://www.goedgepickt.nl/koppeling/myparcel/"
   },
   {
     "name": "fulfilment-software",
     "title": "Fulfilment software",
     "type": "api",
     "documentation": "https://fulfilment-software.com/",
-    "image": "fulfilment.svg"
+    "image": "fulfilment.svg",
+    "url": "https://www.myparcel.nl/koppelingen/fulfilment-software-verzenden/"
   },
   {
     "name": "Afosto",
     "title": "Afosto",
     "type": "api",
     "documentation": "https://afosto.com/nl/docs/apps/myparcel/",
-    "image": "afosto.svg"
+    "image": "afosto.svg",
+    "url": "https://afosto.com/nl/docs/apps/myparcel/"
```

Comment:
Het is hier, maar ook op andere plekken, de bedoeling dat de link achter documentation verwijst naar de documentatie over de MyParcel koppeling en als je op de 'tegel' klikt deze doorlinkt naar de homepage van de betreffende partij. In dit geval dus https://afosto.com